addons/mail/models/mail_message.py

```diff
@@ -141,6 +141,11 @@ def default_get(self, fields):
     mail_ids = fields.One2many('mail.mail', 'mail_message_id', string='Mails', groups="base.group_system")
     canned_response_ids = fields.One2many('mail.shortcode', 'message_ids', string="Canned Responses", store=False)
 
+    @api.depends('author_id')
```

Comment:
might as well remove the `depends` entirely. I also don't think `compute_sudo` matters any.

But I pinged @rco-odoo to see if it woudln't be possible to fix the underlying concern, namely that when we walk up from a dependency we load all records at every level, including the leaf, which can lead to extreme amplification when the root is a very commonly linked record, and the leaf is a very *common* record (e.g. mail.message for which millions isn't an extreme number, or moves).